pkgs/development/python-modules/deezer-py/default.nix

```diff
@@ -0,0 +1,30 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, requests
+, eventlet
+}:
+
+buildPythonPackage rec {
+  pname = "deezer-py";
+  version = "0.0.15";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "2g/GbJ/XMzyT2SDGaBXkf0LZrebvqFzW8yGsEBzK5Gk=";
+  };
+
+  propagatedBuildInputs = [
+    requests
+    eventlet
+  ];
+
+  doCheck = false; # tcp: protocol not found
```

Comment:
```suggestion
  doCheck = false; # tcp: protocol not found
  pythonImportsCheck = [ "deezer" ];
```

Reply:
It fails the same way tests do:
```
File "/nix/store/rdgah0m8hww9pidmjbay2hjf81hnyzmj-python3.8-dnspython-1.16.0/lib/python3.8/site-packages/dns/rdtypes/IN/WKS.py", line 25, in <module>
    _proto_tcp = socket.getprotobyname('tcp')
OSError: protocol not found
```

Reply:
That shouldn't happen. If this is not working the package wont work.

Reply:
But it happens, and the package works.
Why shouldn't it be happening? The build is sandboxed, so it is not surprising it cannot have access to tcp connections.

Reply:
Importing a package should not require a network connection.

Reply:
I agree, it should not, but this is Python, and I guess imports can have side effects…

Reply:
You could try having /etc/protocols in the sandbox and see what happens.

Reply:
Even with [this](https://nixos.wiki/wiki/Packaging/Quirks_and_Caveats#Test_cannot_access_.2Fetc.2Fprotocols.2C_.2Fetc.2Fservices_or_expects_a_special_.2Fetc.2Fpasswd_when_building_in_sandbox) it fails with the same error when importing `eventlet`.